### PR TITLE
api: client: info: add newline after warning

### PR DESCRIPTION
**Pull request by runcom**
Somehow I totally missed it, probably after fixing the first space and force pushing again..

Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: api/client/info.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 
 			// print a warning if devicemapper is using a loopback file
 			if pair[0] == "Data loop file" {
-				fmt.Fprintf(cli.err, " WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.")
+				fmt.Fprintf(cli.err, " WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.\n")
```
Comment (LK4D4) [inline api/client/info.go]: I think you wanted to use `fmt.Fprintln`, because there is no really formatting.

Comment (runcom) [inline api/client/info.go]: thought about that but other warnings use `Fprintf` as well https://github.com/docker/docker/blob/master/api/client/info.go#L109

Should I change all the others as well?

Comment (LK4D4) [inline api/client/info.go]: As you wish :) I just guessed why you forgot newline.

**Comment by runcom** [comment]
@LK4D4 updated! :)

**Comment by MHBauer** [comment]
Guessing, L#36 is the original change, everything else looks good and matches up.

**Comment by thaJeztah** [comment]
Nice, much cleaner

LGTM

thanks for reviewing @MHBauer!

**Comment by calavera** [comment]
LGTM

**Pull request merged**